src/Basque/Mews.Fiscalizations.Basque/Dto/Bizkaia/LROERequest.cs

```diff
@@ -0,0 +1,496 @@
+﻿using System.Xml.Serialization;
+
+namespace Mews.Fiscalizations.Basque.Dto.Bizkaia;
+
+
+// NOTE: Generated code may require at least .NET Framework 4.5 or .NET Core/Standard 2.0.
+/// <remarks/>
+[Serializable()]
+[System.ComponentModel.DesignerCategory("code")]
+[XmlType(AnonymousType = true, Namespace = "https://www.batuz.eus/fitxategiak/batuz/LROE/esquemas/LROE_PJ_240_1_1_FacturasEmi" +
+    "tidas_ConSG_AltaPeticion_V1_0_2.xsd")]
+[XmlRoot(Namespace = "https://www.batuz.eus/fitxategiak/batuz/LROE/esquemas/LROE_PJ_240_1_1_FacturasEmi" +
+    "tidas_ConSG_AltaPeticion_V1_0_2.xsd", IsNullable = false)]
+public class LROEPJ240FacturasEmitidasConSGAltaPeticion
```

Comment:
Let's confirm with the research team that only model 240 is applicable in our case.